### DISCUSSION TITLE
Options - Tranlations and Video resolutions

### DISCUSSION THREAD

**Discussion by andryeltj** [sircharlo/meeting-media-manager#47] [Ideas] [answered]
As the title suggests, include the possibility of translating the app into more languages, and the option to choose the video resolution.

For example, here we use Brazilian Portuguese, with videos at 480p resolution because of internet traffic, and because the brothers have a weak internet.

And that would make using the app easier.

**Comment by sircharlo** [comment, marked as answer]
That's a great idea. I didn't see the need to do that while there were only a handful of users out there, but since translations could potentially help some who now seem to use the app regularly, I think it would be worth it. 
I'll try to have the preliminary groundwork done for this soon. 
As for video resolutions, also a good idea. I'll mull over how to implement this. Possibly a "max resolution" setting.

**Reply by andryeltj** [reply]
Sorry. I forgot to thank you, that this is a huge great application. Congratulations.

Unfortunately for Linux as we don't have a JW Library we have to find out how we can with VLC, so adding features are great for us.

Thank you.

**Reply by sircharlo** [reply]
So, I've made some progress with this! Do you think you could help me out by translating some strings into Portuguese?
[pt.txt](https://github.com/sircharlo/jw-meeting-media-fetcher/files/6542579/pt.txt)

**Reply by andryeltj** [reply]
Yes.
Here is Portuguese Brazilian and Portuguese Portugal:
[pt-br.txt](https://github.com/sircharlo/jw-meeting-media-fetcher/files/6542832/pt-br.txt)
[pt.txt](https://github.com/sircharlo/jw-meeting-media-fetcher/files/6542833/pt.txt)

**Reply by sircharlo** [reply]
Thanks for the strings!
I'll release a new update within a few days with both app translations and a video resolution setting. Stay tuned 😉

**Reply by andryeltj** [reply]
Thanks brother. Receive the Christian love of all those who use your software (and even more of those who use linux).

**Comment by habarahonaa** [comment]
I hope this message finds you well dear brothers! As a macOS user, I really feel the need of an application like this to fetch the media for the meetings. I would like to volunteer to write the locale for Spanish if that’s fine with you.

warm regards and Christian love :)

**Reply by sircharlo** [reply]
Please do brother! It will be much appreciated. You can find the strings in the English strings file; if you attach the Spanish file here once you’re done I’ll include it in the next release 😊

**Reply by sircharlo** [reply]
Or fork my repo and send a pull request 👍🏻

**Comment by sircharlo** [comment]
Hello @habarahonaa! Would you be willing to update the Spanish strings of the app? There have been alot of new strings added these days. If you can't, I understand!
Thanks

**Reply by habarahonaa** [reply]
Hello dear brother! I will gladly update them. Very thankful for your job!

**Reply by habarahonaa** [reply]
So it would be the latest english locale right?

**Reply by sircharlo** [reply]
Thanks for the willingness! Correct
